tests/Doctrine/Tests/DBAL/Schema/MySqlSchemaManagerTest.php

```diff
@@ -23,19 +23,23 @@ class MySqlSchemaManagerTest extends TestCase
 
     protected function setUp() : void
     {
-        $eventManager  = new EventManager();
-        $driverMock    = $this->createMock(Driver::class);
-        $platform      = $this->createMock(MySqlPlatform::class);
+        $eventManager = new EventManager();
+        $driverMock   = $this->createMock(Driver::class);
+
+        $platform = $this->createMock(MySqlPlatform::class);
+        $platform->method('getListTableForeignKeysSQL')
+            ->willReturn('');
```

Comment:
This needs to be mocked because the newly called method has a scalar parameter type hint but the method being mocked doesn't have a return type hint, so PHPUnit cannot mock it correctly. This is to be removed in `master`.